compute_tools/src/http/api.rs

```diff
@@ -49,6 +55,17 @@ async fn routes(req: Request<Body>, compute: &Arc<ComputeNode>) -> Response<Body
             }
         }
 
+        (&Method::GET, "/num_cpus") => {
```

Comment:
It looks way too specific to me, maybe name it `/info`?

Reply:
renamed to `/info`